input variables not supported in task args

Extension Name: vscode-gradle
Extension Version: v3.12.4
OS Version: macOS Monterey
VSCode version: 1.70.2

**Describe the bug**
PR #1035 doesn't seem to have included support for [input variables ](https://code.visualstudio.com/docs/editor/variables-reference#_input-variables)

**To Reproduce**
Define a gradle workflow and reference an input variable from the args, e.g.
```
{
	"version": "2.0.0",
	"tasks": [
		{
			"type": "gradle",
			"script": "app:run",
			"args": "-PfileToRun=${input:fileToRun}",
			...
		}
	],
	"inputs": [
		{
			"type": "pickString",
			"id": "fileToRun",
			"description": "What file do you want to run?",
			"options": [
				"File1",
				"File2"
			]
		},
	]
}
```

A possible build.gradle.kts might include...
```
application {
    // Define the main class for the application.
    mainClass.set("com.example." + project.findProperty("fileToRun").toString() + "Kt")
}
```

**Expected behavior**
the input variable should be dereferenced in the resulting command's arguments

**Output from "Gradle for Java"**
```
> :app:run
Error: Could not find or load main class com.example.${input:fileToRun}Kt
Caused by: java.lang.ClassNotFoundException: com.example.${input:fileToRun}Kt
```

**Does the bug still exist if you disable all other extensions?**
Yes

--- a/extension/src/tasks/taskUtil.ts
+++ b/extension/src/tasks/taskUtil.ts
@@ -170,7 +170,6 @@ export function resolveTaskFromDefinition(
     client: GradleClient
 ): vscode.Task | undefined {
     const taskName = buildTaskName(definition);
-    const args = [definition.script].concat(parseArgsStringToArgv(definition.args.trim())).filter(Boolean);
     const task = new vscode.Task(
         definition,
         workspaceFolder,
@@ -190,8 +189,10 @@ export function resolveTaskFromDefinition(
                     rootProject.getProjectUri().fsPath,
                     definition.script
                 );
-
-                const executeTerminal = new GradleRunnerTerminal(rootProject, args, cancellationKey, client);
+                const resolvedArgs = [resolvedTaskDefinition.script]
+                    .concat(parseArgsStringToArgv(resolvedTaskDefinition.args.trim()))
+                    .filter(Boolean);
+                const executeTerminal = new GradleRunnerTerminal(rootProject, resolvedArgs, cancellationKey, client);
                 executeTerminal.setTask(task);
                 return executeTerminal;
             }

